structure_tests/build.sh

```diff
@@ -26,4 +26,4 @@ fi
 
 envsubst < cloudbuild.yaml.in > cloudbuild.yaml
 cd ..
-gcloud alpha container builds create . --config=structure_tests/cloudbuild.yaml
+gcloud beta container builds submit . --config=structure_tests/cloudbuild.yaml
```

Comment:
We can address those failures as they come up. Making this beta is the better choice here. Our build system has a recent enough version of gcloud that this shouldn't be an issue.